extension/src/popup/views/RecoverAccount/index.tsx

```diff
@@ -218,7 +222,13 @@ export const RecoverAccount = () => {
                     <Button
                       fullWidth
                       isLoading={isSubmitting}
-                      disabled={!(dirty && isValid)}
+                      disabled={
+                        !(
+                          dirty &&
+                          isValid &&
+                          buildMnemonicPhrase(mnemonicPhraseArr).length
```

Comment:
When Recovering using a mnemonic phrase, we need to manually check that the user has entered a phrase bc these are custom fields now, not Formik